app/controllers/partners/families_controller.rb

```diff
@@ -98,5 +102,14 @@ def sort_direction
     def sort_column
       Family.column_names.include?(params[:sort]) ? params[:sort] : "guardian_last_name"
     end
+
+    def archive_children
+      service = ArchiveFamilyChildren.new(family: @family)
```

Comment:
I feel like this probably should move to a more general `UpdateFamily` service. But I'm not sure it's worth the additional work due to the small footprint this feature has on the app.